<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSJDemoQT_Matrix</class>
 <widget class="QDialog" name="KSJDemoQT_Matrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KSJDemoQT</string>
  </property>
  <widget class="QWidget" name="PREVIEWWND_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>501</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DEVICENUM_Label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Device Num:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DEVICE_LIST_ComboBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="ExposureLinesSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="GainSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure Time(ms):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure Lines:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ExposureTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range(ms):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="EXPOSURE_TIME_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="EXPOSURE_LINES_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="GAIN_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="EXPOSURE_TIME_REAL_Label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>REAL EXP TIME</string>
   </property>
  </widget>
  <widget class="QLabel" name="FOV_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>EXP LINES RANGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FOV Range:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Preview</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Col Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Col Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MultiFrame:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetPreviewFovButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Preview Fov</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="PreviewColStartSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="PreviewColSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="PreviewRowStartSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="PreviewRowSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="PreviewMultiFrameSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Capture</string>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Col Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Col Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MultiFrame:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetCaptureFovButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Capture Fov</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureColStartSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureColSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureRowStartSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureRowSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureMultiFrameSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PreviewButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CaptureButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="ERROR_INFO_Label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>530</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Error Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="ELAPSE_TIME_Label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>530</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Elapse</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="FunctionTableWidget">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>0</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="SaveCheckBox">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>450</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="DEVICENUM_Label_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>WB Mode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="WBComboBox">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>190</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="PresettingComboBox">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>220</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DEVICENUM_Label_3">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>220</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Presetting:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>250</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Phi:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="PhiSpinBox">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>250</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="RSlider">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>280</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>R:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="REdit">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>280</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="GSlider">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>310</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>310</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>G:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="GEdit">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="BSlider">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>340</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>340</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BEdit">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>340</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="CCMComboBox">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>370</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DEVICENUM_Label_4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>370</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CCM Mode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CCMPresettingComboBox">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>400</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DEVICENUM_Label_5">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>400</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Presetting:</string>
   </property>
  </widget>
  <widget class="QSlider" name="GSlider00">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>450</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="GEdit00">
   <property name="geometry">
    <rect>
     <x>979</x>
     <y>450</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="GEdit01">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>450</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="GSlider01">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>450</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="GEdit02">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>450</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="GSlider02">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>450</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="GSlider12">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>480</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="GEdit10">
   <property name="geometry">
    <rect>
     <x>979</x>
     <y>480</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="GSlider10">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>480</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="GSlider11">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>480</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="GEdit11">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>480</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="GEdit12">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>480</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="GEdit20">
   <property name="geometry">
    <rect>
     <x>979</x>
     <y>510</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="GSlider20">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>510</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="GEdit21">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>510</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="GSlider22">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>510</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="GEdit22">
   <property name="geometry">
    <rect>
     <x>1159</x>
     <y>510</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="GSlider21">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>510</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
